plugins/UM3NetworkPrinting/src/Cloud/CloudApiClient.py

```diff
@@ -159,9 +159,13 @@ def _addCallback(self,
                      model: Type[CloudApiClientModel],
                      ) -> None:
         def parse() -> None:
+            # Don't try to parse the reply if we didn't get one
+            if reply.attribute(QNetworkRequest.HttpStatusCodeAttribute) is None:
```

Comment:
We might wanna log at least something at the debug level that we didn't get a reply? Because this is never expected behaviour.